client/app/scripts/superdesk-authoring/metadata/views/metadata-tags.html

```diff
@@ -0,0 +1,11 @@
+<div class="dropdown dropright dropdown-bigger dropdown-terms">
+    <input class="line-input" type="text" ng-show="adding" ng-model="newTag" ng-disabled="disabled" ng-keyup="key($event)">
+    <button class="dropdown-toggle" ng-show="!adding" ng-disabled="disabled" ng-click="adding = true"><i class="icon-white icon-plus-large"></i></button>
+</div>
+
+<div class="terms">
+    <ul>
+        <li ng-if="!disabled" ng-class="{selected: isSelected(t)}" ng-repeat="t in tags track by t" ng-click="toggle(t)">{{ t }}</li>
```

Comment:
imo it would be better to handle disabled state in `toggle` func to avoid this duplication